hrm-domain/hrm-core/profile/profileService.ts

```diff
@@ -351,15 +351,16 @@ export const createProfileSection = async (
   { user }: { user: TwilioUser },
 ): Promise<profileDB.ProfileSection> => {
   const { content, profileId, sectionType } = payload;
-  const section = profileDB.createProfileSection()(accountSid, {
+  const section = await profileDB.createProfileSection()(accountSid, {
     content,
     profileId,
     sectionType,
     createdBy: user.workerSid,
   });
-  getProfileById()(accountSid, profileId, true).then(profile =>
-    notifyUpdateProfile({ accountSid, profile }),
-  );
+  getProfileById()(accountSid, profileId, true).then(profile => {
+    console.log('Broadcasting profile', JSON.stringify(profile, null, 2));
+    notifyUpdateProfile({ accountSid, profile });
+  });
```

Comment:
Wouldn't this be better abstracted in `doProfileChangeNotification`, just like `doContactChangeNotification`/`doCaseChangeNotification` (where it expects the entity id and performs the lookup in DB within the notify handler)?. Besides "DRY", it makes it impossible to send outdated data.